benchmarks/wrappers/cloudflare/python/container/nosql.py

```
@@ -0,0 +1,117 @@
+        """Set worker URL for NoSQL proxy (called by handler)"""
+        nosql.worker_url = url
+
+    def _make_request(self, operation: str, params: dict) -> dict:
+        """Make HTTP request to worker nosql proxy"""
+        if not nosql.worker_url:
+            raise RuntimeError("Worker URL not set - cannot access NoSQL")
+        
+        url = f"{nosql.worker_url}/nosql/{operation}"
+        data = json.dumps(params).encode('utf-8')
+        
+        req = urllib.request.Request(url, data=data, method='POST')
```

Ruff (0.15.6)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)